README.md

```diff
@@ -805,6 +805,38 @@ struct FetchMoviesPhaseAtom: ValueAtom, Refreshable, Hashable {
 }
 ```
 
+
+#### [Resettable](https://ra1028.github.io/swiftui-atom-properties/documentation/atoms/resettable)
+
+`Resettable` allows you to implement additional reset behavior to an atom.
+
+<details><summary><code>📖 Expand to see example</code></summary>
+
+It adds custom reset behavior to an Atom that will be executed prior to new cache generation.
+
+It's useful when need to handle reset side-effects or when utilizing private atom from other atoms.
+  
+In following example, `RandomIntAtom` generates a random value using private `RandomNumberGeneratorAtom`, and `Resettable` gives ability to reset `RandomNumberGeneratorAtom` with new randomizer seed.
+
+```swift
+struct RandomIntAtom: ValueAtom, Resettable, Hashable {
+    func value(context: Context) -> Int {
+        let generator = context.watch(RandomNumberGeneratorAtom())
```

Comment:
Not tested, but tried to expand existing example not to introduce completely new context. Used [this](https://developer.apple.com/documentation/swift/systemrandomnumbergenerator) as reference